distance_resolution: 0.005
dsr_toffet: 3.23
firing_frequency: 0.018
rx: 0.03615
ry: -0.017
rz: 0
min_distance: 1
max_distance: 200
echo_mode: 2
num_lasers: 128
lasers:
- {laser_id: 0, vert_angle: -1356, hori_angle: 595}
- {laser_id: 1, vert_angle: -109, hori_angle: 425}
- {laser_id: 2, vert_angle: -439, hori_angle: 255}
- {laser_id: 3, vert_angle: 191, hori_angle: 85}
- {laser_id: 4, vert_angle: -665, hori_angle: 595}
- {laser_id: 5, vert_angle: -29, hori_angle: 425}
- {laser_id: 6, vert_angle: -359, hori_angle: 255}
- {laser_id: 7, vert_angle: 271, hori_angle: 85}
- {laser_id: 8, vert_angle: -579, hori_angle: 595}
- {laser_id: 9, vert_angle: 51, hori_angle: 425}
- {laser_id: 10, vert_angle: -279, hori_angle: 255}
- {laser_id: 11, vert_angle: 351, hori_angle: 85}
- {laser_id: 12, vert_angle: -498, hori_angle: 595}
- {laser_id: 13, vert_angle: 131, hori_angle: 425}
- {laser_id: 14, vert_angle: -199, hori_angle: 255}
- {laser_id: 15, vert_angle: 506, hori_angle: 85}
- {laser_id: 16, vert_angle: -419, hori_angle: 595}
- {laser_id: 17, vert_angle: 210, hori_angle: 425}
- {laser_id: 18, vert_angle: -1958, hori_angle: 255}
- {laser_id: 19, vert_angle: -129, hori_angle: 85}
- {laser_id: 20, vert_angle: -339, hori_angle: 595}
- {laser_id: 21, vert_angle: 291, hori_angle: 425}
- {laser_id: 22, vert_angle: -715, hori_angle: 255}
- {laser_id: 23, vert_angle: -49, hori_angle: 85}
- {laser_id: 24, vert_angle: -259, hori_angle: 595}
- {laser_id: 25, vert_angle: 371, hori_angle: 425}
- {laser_id: 26, vert_angle: -599, hori_angle: 255}
- {laser_id: 27, vert_angle: 31, hori_angle: 85}
- {laser_id: 28, vert_angle: -179, hori_angle: 595}
- {laser_id: 29, vert_angle: 596, hori_angle: 425}
- {laser_id: 30, vert_angle: -519, hori_angle: 255}
- {laser_id: 31, vert_angle: 111, hori_angle: 85}
- {laser_id: 32, vert_angle: -99, hori_angle: 595}
- {laser_id: 33, vert_angle: -429, hori_angle: 425}
- {laser_id: 34, vert_angle: 201, hori_angle: 255}
- {laser_id: 35, vert_angle: -2500, hori_angle: 85}
- {laser_id: 36, vert_angle: -19, hori_angle: 595}
- {laser_id: 37, vert_angle: -349, hori_angle: 425}
- {laser_id: 38, vert_angle: 281, hori_angle: 255}
- {laser_id: 39, vert_angle: -765, hori_angle: 85}
- {laser_id: 40, vert_angle: 61, hori_angle: 595}
- {laser_id: 41, vert_angle: -269, hori_angle: 425}
- {laser_id: 42, vert_angle: 361, hori_angle: 255}
- {laser_id: 43, vert_angle: -609, hori_angle: 85}
- {laser_id: 44, vert_angle: 141, hori_angle: 595}
- {laser_id: 45, vert_angle: -189, hori_angle: 425}
- {laser_id: 46, vert_angle: 546, hori_angle: 255}
- {laser_id: 47, vert_angle: -529, hori_angle: 85}
- {laser_id: 48, vert_angle: 221, hori_angle: 595}
- {laser_id: 49, vert_angle: -1604, hori_angle: 425}
- {laser_id: 50, vert_angle: -119, hori_angle: 255}
- {laser_id: 51, vert_angle: -448, hori_angle: 85}
- {laser_id: 52, vert_angle: 301, hori_angle: 595}
- {laser_id: 53, vert_angle: -685, hori_angle: 425}
- {laser_id: 54, vert_angle: -39, hori_angle: 255}
- {laser_id: 55, vert_angle: -369, hori_angle: 85}
- {laser_id: 56, vert_angle: 381, hori_angle: 595}
- {laser_id: 57, vert_angle: -589, hori_angle: 425}
- {laser_id: 58, vert_angle: 41, hori_angle: 255}
- {laser_id: 59, vert_angle: -289, hori_angle: 85}
- {laser_id: 60, vert_angle: 656, hori_angle: 595}
- {laser_id: 61, vert_angle: -509, hori_angle: 425}
- {laser_id: 62, vert_angle: 121, hori_angle: 255}
- {laser_id: 63, vert_angle: -208, hori_angle: 85}
- {laser_id: 64, vert_angle: -835, hori_angle: -85}
- {laser_id: 65, vert_angle: -69, hori_angle: -255}
- {laser_id: 66, vert_angle: -399, hori_angle: -425}
- {laser_id: 67, vert_angle: 231, hori_angle: -595}
- {laser_id: 68, vert_angle: -619, hori_angle: -85}
- {laser_id: 69, vert_angle: 11, hori_angle: -255}
- {laser_id: 70, vert_angle: -319, hori_angle: -425}
- {laser_id: 71, vert_angle: 311, hori_angle: -595}
- {laser_id: 72, vert_angle: -539, hori_angle: -85}
- {laser_id: 73, vert_angle: 91, hori_angle: -255}
- {laser_id: 74, vert_angle: -239, hori_angle: -425}
- {laser_id: 75, vert_angle: 396, hori_angle: -595}
- {laser_id: 76, vert_angle: -459, hori_angle: -85}
- {laser_id: 77, vert_angle: 171, hori_angle: -255}
- {laser_id: 78, vert_angle: -159, hori_angle: -425}
- {laser_id: 79, vert_angle: 741, hori_angle: -595}
- {laser_id: 80, vert_angle: -379, hori_angle: -85}
- {laser_id: 81, vert_angle: 251, hori_angle: -255}
- {laser_id: 82, vert_angle: -1034, hori_angle: -425}
- {laser_id: 83, vert_angle: -89, hori_angle: -595}
- {laser_id: 84, vert_angle: -299, hori_angle: -85}
- {laser_id: 85, vert_angle: 331, hori_angle: -255}
- {laser_id: 86, vert_angle: -639, hori_angle: -425}
- {laser_id: 87, vert_angle: -9, hori_angle: -595}
- {laser_id: 88, vert_angle: -219, hori_angle: -85}
- {laser_id: 89, vert_angle: 441, hori_angle: -255}
- {laser_id: 90, vert_angle: -559, hori_angle: -425}
- {laser_id: 91, vert_angle: 71, hori_angle: -595}
- {laser_id: 92, vert_angle: -139, hori_angle: -85}
- {laser_id: 93, vert_angle: 1150, hori_angle: -255}
- {laser_id: 94, vert_angle: -479, hori_angle: -425}
- {laser_id: 95, vert_angle: 151, hori_angle: -595}
- {laser_id: 96, vert_angle: -58, hori_angle: -85}
- {laser_id: 97, vert_angle: -389, hori_angle: -255}
- {laser_id: 98, vert_angle: 241, hori_angle: -425}
- {laser_id: 99, vert_angle: -1174, hori_angle: -595}
- {laser_id: 100, vert_angle: 21, hori_angle: -85}
- {laser_id: 101, vert_angle: -309, hori_angle: -255}
- {laser_id: 102, vert_angle: 321, hori_angle: -425}
- {laser_id: 103, vert_angle: -650, hori_angle: -595}
- {laser_id: 104, vert_angle: 101, hori_angle: -85}
- {laser_id: 105, vert_angle: -229, hori_angle: -255}
- {laser_id: 106, vert_angle: 416, hori_angle: -425}
- {laser_id: 107, vert_angle: -569, hori_angle: -595}
- {laser_id: 108, vert_angle: 181, hori_angle: -85}
- {laser_id: 109, vert_angle: -149, hori_angle: -255}
- {laser_id: 110, vert_angle: 900, hori_angle: -425}
- {laser_id: 111, vert_angle: -489, hori_angle: -595}
- {laser_id: 112, vert_angle: 261, hori_angle: -85}
- {laser_id: 113, vert_angle: -924, hori_angle: -255}
- {laser_id: 114, vert_angle: -79, hori_angle: -425}
- {laser_id: 115, vert_angle: -409, hori_angle: -595}
- {laser_id: 116, vert_angle: 341, hori_angle: -85}
- {laser_id: 117, vert_angle: -629, hori_angle: -255}
- {laser_id: 118, vert_angle: 1, hori_angle: -425}
- {laser_id: 119, vert_angle: -329, hori_angle: -595}
- {laser_id: 120, vert_angle: 471, hori_angle: -85}
- {laser_id: 121, vert_angle: -549, hori_angle: -255}
- {laser_id: 122, vert_angle: 81, hori_angle: -425}
- {laser_id: 123, vert_angle: -249, hori_angle: -595}
- {laser_id: 124, vert_angle: 1500, hori_angle: -85}
- {laser_id: 125, vert_angle: -469, hori_angle: -255}
- {laser_id: 126, vert_angle: 161, hori_angle: -425}
- {laser_id: 127, vert_angle: -169, hori_angle: -595}
